src/Libraries/UnitsUI/UnitsUI.cs

```diff
@@ -128,6 +130,10 @@ public double Value
             }
         }
 
+        public MeasurementInputBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts):base(inPorts, outPorts) { }
```

Comment:
API Change : Public constructors have been added to `MeasurementInputBase` to facilitate serialization.